index.html

```diff
@@ -1166,14 +1166,17 @@ <h3>Contexts</h3>
         <h3>Identifiers</h3>
 
         <p>
-When expressing statements about a specific thing, such as a person, product,
-or organization, it is often useful to use some kind of identifier so that
-others can express statements about the same thing. This specification defines
-the optional <code>id</code> <a>property</a> for such identifiers. The
-<code>id</code> <a>property</a> is intended to unambiguously refer to an object,
-such as a person, product, or organization. Using the <code>id</code>
-<a>property</a> allows for the expression of statements about specific things in
-the <a>verifiable credential</a>.
+When expressing statements about a specific thing, such as a person, product, or
+organization, it can be useful to use a globally unique identifier for that thing.
+Globally unique identifiers enable others to express statements
+about the same thing. This specification defines the optional <code>id</code>
+<a>property</a> for such identifiers. The <code>id</code> <a>property</a>
+allows for the expression of statements about specific things in the
+<a>verifiable credential</a> and is set by an <a>issuer</a> when expressing
+objects in a <a>verifiable credential</a> or a <a>holder</a> when expressing
+objects in a <a>verifiable presentation</a>. Example <code>id</code> values
+include UUIDs (`urn:uuid:0c07c1ce-57cb-41af-bef2-1b932b986873`), HTTP URLs
+(`https://id.example/123`), and DIDs (`did:example:1234abcd`).
```

Comment:
```suggestion
(`https://id.example/people#alice`), and DIDs (`did:example:1234abcd`).
```

@TallTed I'm making the change above in order to avoid another HTTP Range-14 perma-thread. :)